src/citrine/builders/auto_configure.py

```diff
@@ -399,6 +400,11 @@ def from_material(
         print_status_info: bool
             Whether to print the status info during validation of assets.
             Default: False
+        prefer_valid: Boolean
+            If True, this sets the filterSparse option to True and sets targetMaxModels and
```

Comment:
I think this comment needs to be adjusted. The `filterSparse` and `targetMaxModels` parameters are never user exposed, and a user has no knowledge of "predictor build settings".

Perhaps something along the lines of:
`"If True, enables filtering of sparse descriptors and trimming of excess graph components in attempt to return a default configuration that will pass validation."`

Reply:
That's a good point, I copied the prefer_valid feature flag description.  I'll use your suggestion.